# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

DoorDetectorActionGoal action_goal
DoorDetectorActionResult action_result
DoorDetectorActionFeedback action_feedback
